### PR TITLE
feat: add header menu component AB#18868

### PR DESCRIPTION
**Pull request by agnesesaulite**

### CODE REVIEW

**Review by elwinschmitz — CHANGES_REQUESTED** [review]

File: interfaces/HO-Portal/src/app/components/header/header.component.scss [hunk 1]
```diff
@@ -1,6 +1,45 @@
 ion-header {
   border-bottom: 1px solid var(--ion-color-primary);
+
   &::after {
     background-image: none; //hide ion-header drop shadow
   }
 }
+
+.pop-over-container {
+  background: #ffffff;
```
[the remark this answers is not in the document]
Comment (agnesesaulite) [reply interfaces/HO-Portal/src/app/components/header/header.component.scss:10]: @elwinschmitz Where are variables defined?

File: interfaces/HO-Portal/src/app/components/header/header.component.html [hunk 2]
```diff
@@ -6,43 +6,78 @@
...
+                    ></ion-icon>
+                  </ion-button>
+                </ion-list-header>
+                <ion-item
+                  lines="full"
+                  [title]="'header.home' | translate"
+                  [attr.aria-label]="'header.home' | translate"
+                  [routerLink]="['/', appRoute.home]"
+                >
+                  <ion-label>Home</ion-label>
```
[the remark this answers is not in the document]
Comment (agnesesaulite) [reply interfaces/HO-Portal/src/app/components/header/header.component.html]: Thanks! Done!

File: interfaces/HO-Portal/src/app/components/header/header.component.html [hunk 3]
```diff
@@ -6,43 +6,78 @@
...
+                </ion-item>
+                <ion-item
+                  lines="full"
+                  [title]="'header.users' | translate"
+                  [attr.aria-label]="'header.users' | translate"
+                  [routerLink]="['/', appRoute.users]"
+                >
+                  <ion-label>Users</ion-label>
```
[the remark this answers is not in the document]
Comment (agnesesaulite) [reply interfaces/HO-Portal/src/app/components/header/header.component.html]: As well.

File: interfaces/HO-Portal/src/app/components/header/header.component.html [hunk 4]
```diff
@@ -6,43 +6,78 @@
...
+                </ion-item>
+                <ion-item
+                  lines="full"
+                  [title]="'header.help' | translate"
+                  [attr.aria-label]="'header.help' | translate"
+                  [routerLink]="['/', appRoute.help]"
+                >
+                  <ion-label>Help</ion-label>
```
[the remark this answers is not in the document]
Comment (agnesesaulite) [reply interfaces/HO-Portal/src/app/components/header/header.component.html]: As well.

File: interfaces/HO-Portal/src/app/components/header/header.component.html [hunk 5]
```diff
@@ -6,43 +6,78 @@
...
+            </ion-content>
+          </ng-template>
+        </ion-popover>
       </ion-col>
       <ion-col
         *ngIf="!isIframeHeader"
         size="4"
       >
-        <h1 class="ion-no-margin">
-          {{ title }}
```
[the remark this answers is not in the document]
Comment (agnesesaulite) [reply interfaces/HO-Portal/src/app/components/header/header.component.html]: Added back.

File: interfaces/HO-Portal/src/app/components/header/header.component.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (agnesesaulite) [reply interfaces/HO-Portal/src/app/components/header/header.component.scss]: fixed

**Review by kaspars-grinbergs-accenture — APPROVED** [review]

**Review by elwinschmitz — APPROVED** [review]

**Pull request merged**